chainer/training/extensions/multistep_shift.py

```diff
@@ -0,0 +1,63 @@
+from __future__ import division
+
+from chainer.training import extension
+
+
+class MultistepShift(extension.Extension):
+
+    """Trainer extension to shift an optimizer attribute in several steps.
+
+    This extension changes an optimizer attribute in several steps, every step
+    the attribute will multiply a factor ``gamma``.
+
+    For example, suppose that this extension is called at every iteration,
+    and ``init = x``, ``gamma = y``, ``step_value = [s1, s2, s3]``.
+    Then during the iterations from 0 to s1, the attr will be ``x``. During the
```

Comment:
The docs is a bit ambiguous because it is unclear if `0 to s1` is inclusive or not.
How about changing it to  `0 to (s1 - 1)`?